<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeasureSelectedFeaturesBase</class>
 <widget class="QDockWidget" name="QgsMeasureSelectedFeaturesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measure Selected Features</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="mTotalLabel">
      <property name="text">
       <string>total</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mTotalLineEdit">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mConvertedLabel">
      <property name="text">
       <string>Converted</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="mKilometersLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="mAreaHectaresLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
